glean-core/src/metrics/timespan.rs

```diff
@@ -108,7 +108,12 @@ impl TimespanMetric {
         }
 
         if self.start_time.is_some() {
-            log::warn!("Timespan already running. Raw value not recorded.");
+            record_error(
+                glean,
+                &self.meta,
+                ErrorType::InvalidValue,
```

Comment:
One could argue this isn't "Invalid Value" but is more like "Invalid State".  But that's a question for another bug...

Reply:
Yup, let's file a new bug for this ;)

Reply:
Done: https://bugzilla.mozilla.org/show_bug.cgi?id=1566380